config/routes.rb

```diff
@@ -1,10 +1,14 @@
 Rails.application.routes.draw do
-  get '/healthcheck', :to => proc { [200, {}, ['OK']] }
+  with_options format: false do |r|
+    r.get '/healthcheck', :to => proc { [200, {}, ['OK']] }
 
-  put "/draft-content/*base_path", to: "content_items#put_draft_content_item"
-  put "/content/*base_path", to: "content_items#put_live_content_item"
+    r.constraints base_path: %r[/.*] do
+      put "/draft-content/*base_path", to: "content_items#put_draft_content_item"
+      put "/content/*base_path", to: "content_items#put_live_content_item"
```

Comment:
You've now got an extra `/` in here.  Because `*base_path` is constrained to start with a `/`, the extra `/` in the route here isn't required.

Due to the logic that's now present here, it's probably worth adding some routing specs (eg https://github.com/alphagov/content-store/blob/master/spec/routing/content_item_routing_spec.rb)